modules/account-lib/package.json

```diff
@@ -41,6 +42,8 @@
     "@ethereumjs/common": "^2.4.0",
     "@ethereumjs/tx": "^3.3.0",
     "@hashgraph/sdk": "^2.0.17",
+    "@polkadot/api": "^6.6.1",
```

Comment:
Where are you using this dependency?

Reply:
gives us @polkadot/util-crypto @polkadot/types @polkadot/util which we use